vscode/src/jsonrpc/agent-protocol.ts

```diff
@@ -180,6 +176,15 @@ export type ClientRequests = {
         },
     ]
 
+    // Gets whether the specified URI is sensitive and should not be sent to
+    // LLM providers.
+    'ignore/forUri': [{ uri: string }, IgnoreForUriResult]
```

Comment:
nit: I generally try to use verbs in the second part of the RPC method name, and use nouns only if it's very awkward to use a noun. I think `codyignore` is also a bit easier to read since it's a noun.

```suggestion
    'codyignore/test': [{ uri: string }, IgnoreForUriResult]
```

Also, it should be fine to use an anonymous literal `{ policy: 'ignore' | 'use' }` here instead of `IgnoreForUriResult`. If it didn't work then I can look into fixing the bindings generator.